launch-slave

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -x
+
+DOCKER_SOCKET=/var/run/docker.sock
+DOCKER_GROUP=docker
+REGULAR_USER=jenkins
+
+if [ -S ${DOCKER_SOCKET} ]; then
+    DOCKER_GID=$(stat -c '%g' ${DOCKER_SOCKET})
+    groupadd -for -g ${DOCKER_GID} ${DOCKER_GROUP}
+    usermod -aG ${DOCKER_GROUP} ${REGULAR_USER}
```

Comment:
I guess you can remove this line and use --group-add docker instead in the command line

Reply:
maybe but this one work (: